Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityArticulatedHand.cs

```diff
@@ -0,0 +1,491 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Input;
+
+#if UNITY_WSA
+using UnityEngine;
+using UnityEngine.XR.WSA.Input;
+#endif
+
+#if WINDOWS_UWP
+using System;
+using System.Collections.Generic;
+using Windows.Perception;
+using Windows.Perception.People;
+using Windows.UI.Input.Spatial;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
+{
+    /// <summary>
+    /// A Windows Mixed Reality Controller Instance.
+    /// </summary>
+    [MixedRealityController(
+        SupportedControllerType.ArticulatedHand,
+        new[] { Handedness.Left, Handedness.Right })]
+    public class WindowsMixedRealityArticulatedHand : WindowsMixedRealityController, IMixedRealityHand
```

Comment:
I don't know if I fundamentally agree with this inheritance choice. If windows mixed reality controller is truly for controllers compared to just being a confusing input term, we may long term want to have articulated hands not directly inherit from WindowsMixedRealityController

Can we get an issue logging this if this review needs to go in without this change?

Reply:
Can you file an issue for this with your background/question? I don't think that I have sufficient background here to be able to fully argue your case.

Reply:
As long as the change (if truly needed) does not constitute a breaking change, we can definitly defer unitl after RC1.

We **could** take this merge and fix issues in the developent branch prior to declaring RC1 for better visibility on the scope of the change.